dmsgpty/const_windows.go

```diff
@@ -0,0 +1,14 @@
+//+build windows
+
+package dmsgpty
+
+// Constants related to CLI.
+const (
+	DefaultCLINet = "unix"
+)
```

Comment:
I'm not familiar with **dmsg** at all, and I didn't understand why set `unix` as `DefaultCLINet` value.

Reply:
we are using `unix socket` in both windows and *nix